source/extensions/access_loggers/open_telemetry/access_log_impl.cc

```diff
@@ -102,6 +102,8 @@ void AccessLog::emitLog(const Formatter::HttpFormatterContext& log_context,
     auto trace_id = absl::StrCat(Hex::uint64ToHex(0), trace_id_hex);
     *log_entry.mutable_trace_id() = absl::HexStringToBytes(trace_id);
   }
+  std::string span_id_hex = log_context.activeSpan().getSpanIdAsHex();
```

Comment:
nit: const

Reply:
I fixed this issue [here](https://github.com/envoyproxy/envoy/pull/33909/commits/2f9767d7f6f7edbbcacd148a2efc31953b87df2b#diff-558f961772efddc0955235c7588402f93ee2691724bc833ead18acfac6d0b83fL105).